groovy/postInit/chemistry/ChemistryOverhaul.groovy

```diff
@@ -3580,18 +3599,30 @@ LCR.recipeBuilder()
         .EUt(30)
         .buildAndRegister()
 
+LCR.recipeBuilder()
```

Comment:
this recipe is not stoichiometrically balanced, remove it

Reply:
Actually, all of this makes no sense, EDTA is not soluble in water. Remove both of these recipes and replace it with this:

LCR.recipeBuilder()
        .inputs(ore('dustSodiumCyanide') * 12)
        .fluidInputs(fluid('ethylenediamine') * 1000)
        .fluidInputs(fluid('formaldehyde') * 4000)
        .fluidInputs(fluid('hydrochloric_acid') * 4000)
        .outputs(metaitem('dustEthylenediaminetetraaceticAcid') * 36)
        .fluidOutputs(fluid('ammonia') * 4000)
        .fluidOutputs(fluid('wastewater') * 4000)
        .duration(300)
        .EUt(30)
        .buildAndRegister()
       
BR.recipeBuilder()
        .inputs(ore('dustEthylenediaminetetraaceticAcid') * 36)
        .fluidInputs(fluid('sodium_hydroxide_solution') * 4000)
        .fluidOutputs(fluid('ethylenediaminetetraacetic_acid_solution') * 4000)
        .duration(300)
        .EUt(30)
        .buildAndRegister()
       
(you will have to redo these materials as well)

Reply:
done
materials shouldn't need to be done considering H4EDTA dust is already registered